serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -85,9 +85,9 @@ Knative CLI::
 +
 --
 .Create a project using Knative CLI
-[source,shell]
+[source,shell,subs="attributes"]
```

Comment:
the sbus=attr is not needed unless you need some env to be replaced.

Reply:
Done.